ktor-shared/ktor-websocket-serialization/common/src/WebsocketChannelSerialization.kt

```diff
@@ -0,0 +1,78 @@
+import io.ktor.http.cio.websocket.*
```

Comment:
package is empty

Reply:
Also, why not to put this inside `serialization` module, IMO, there is no sense to create additional dependency for this one file

Reply:
I mean, there is no sense to split `WebsocketChannelSerialization` from `WebsocketContentConverter`

Reply:
Websockets are an optional dependency. To put it to `ktor-serialization `, we need to make it dependant on `ktor-server-websockets`. Right now there is nothing used from websockets themselves, but it's better to keep them separated for better backwards compatibility in the future.

Reply:
No, you don't need to add new dependencies.
websockets API (so, mainly `WebSocketSession`) is in `ktor-http:ktor-http-cio`
And, as now `ktor-serialization` depends on it to introduce `WebsocketContentConverter`, I really don't understand, why we need one new module (`ktor-websoket-serialization`), which depends only on `ktor-serialization`.